cinema/views.py

```diff
@@ -31,8 +41,29 @@ class CinemaHallViewSet(viewsets.ModelViewSet):
 
 
 class MovieViewSet(viewsets.ModelViewSet):
-    queryset = Movie.objects.all()
-    serializer_class = MovieSerializer
+    queryset = Movie.objects.prefetch_related("genres", "actors")
+
+    @staticmethod
+    def params_to_ints(query_string):
+        return [int(num) for num in query_string.split(",")]
```

Comment:
The `params_to_ints` method should include error handling to gracefully handle non-integer values in the query string. Consider using a try-except block to catch `ValueError` exceptions.